shell/browser/api/electron_api_web_contents.cc

```diff
@@ -3028,20 +3022,17 @@ void WebContents::Print(gin::Arguments* args) {
     settings.Set(printing::kSettingMarginsType, static_cast<int>(margin_type));
 
     if (margin_type == printing::mojom::MarginType::kCustomMargins) {
-      base::Value::Dict custom_margins;
-      int top = 0;
-      margins.Get("top", &top);
-      custom_margins.Set(printing::kSettingMarginTop, top);
-      int bottom = 0;
-      margins.Get("bottom", &bottom);
-      custom_margins.Set(printing::kSettingMarginBottom, bottom);
-      int left = 0;
-      margins.Get("left", &left);
-      custom_margins.Set(printing::kSettingMarginLeft, left);
-      int right = 0;
-      margins.Get("right", &right);
-      custom_margins.Set(printing::kSettingMarginRight, right);
-      settings.Set(printing::kSettingMarginsCustom, std::move(custom_margins));
+      const int top = margins.ValueOrDefault("top", 0);
+      const int bottom = margins.ValueOrDefault("bottom", 0);
+      const int left = margins.ValueOrDefault("left", 0);
+      const int right = margins.ValueOrDefault("right", 0);
+
+      settings.Set(printing::kSettingMarginsCustom,
+                   base::Value::Dict{}
+                       .Set(printing::kSettingMarginTop, top)
```

Comment:
any reason to not inline top, left, bottom and right here ?

Reply:
Wrapped in 2df717b22f871a73e9be38b82a8efd8e1c5b502f